content/en/docs/concepts/signals/traces.md

```diff
@@ -90,106 +93,111 @@ Sample Trace:
-
+ 
+A key Trace component in OpenTelemetry is a Span:
+ 
+A [**Span**](/docs/concepts/observability-primer/#spans) represents a unit of
+work or operation. Spans are the building blocks of Traces.
+ 
+We'll dive into Spans in [greater
+detail](/docs/concepts/observability-primer/#spans), but let's first look at the
+list of OpenTelemetry components that play a part in instrumenting our code:
+ 
 - Tracer
 - Tracer Provider
 - Trace Exporter
 - Trace Context
-
-### Tracer Provider
-
-A Tracer Provider (sometimes called `TracerProvider`) is a factory for
-`Tracer`s. In most applications, a Tracer Provider is initialized once and its
-lifecycle matches the application's lifecycle. Tracer Provider initialization
-also includes Resource and Exporter initialization. It is typically the first
-step in tracing with OpenTelemetry. In some language SDKs, a global Tracer
-Provider is already initialized for you.
-
+ 
 ### Tracer
-
-A Tracer creates spans containing more information about what is happening for a
-given operation, such as a request in a service. Tracers are created from Tracer
-Providers. In some languages, a global Tracer is already initialized for you.
-
+ 
+A `Tracer` creates spans, where each span has information about what's happening
+for a given operation; for example a request in a service. Tracers are created
+from `TracerProvider`s. In some languages, a global Tracer is already
+initialized for you.
+ 
+### Tracer Provider
+ 
+A **Tracer Provider** (sometimes called `TracerProvider`) is a factory to create
+Tracers. In most applications, a Tracer Provider is initialized once, and the
+Trace Provider's lifecycle matches the application's lifecycle.
+ 
+Initializing a Tracer Provider is typically the first step in tracing with
+OpenTelemetry. The Tracer Provider's initialization also includes Resource and
+Exporter initialization. In some language SDKs, a global Tracer Provider is
+already initialized for you.
+ 
 ### Trace Exporters
-
-Trace Exporters send traces to a consumer. This consumer can be standard output
-for debugging and development-time, the OpenTelemetry Collector, or any
-open-source or vendor backend of your choice.
-
+ 
+**Trace Exporters** send traces to a consumer. The consumer can be standard output
+(std.out) for debugging and development-time, the OpenTelemetry Collector, or
+any open-source or vendor backend of your choice.
+ 
 ### Trace Context
-
-Trace Context is metadata about trace spans that provides correlation between
-spans across service and process boundaries. For example, let's say that Service
-A calls Service B and you want to track the call in a trace. In that case,
-OpenTelemetry will use Trace Context to capture the ID of the trace and current
-span from Service A, so that spans created in Service B can connect and add to
+ 
+The **Trace Context** is metadata about trace **Spans**. It provides correlation between
+Spans, across service and process boundaries. For example, let's say that
```

Comment:
```suggestion
A **trace context** is data about trace spans. It provides correlation between
spans, across service and process boundaries. For example, let's say that
```

Reply:
@cartermp @melbgirl et al.: note the suggested change of "metadata" to just "data". Does that make sense to you?

Reply:
That makes sense to me.

Reply:
Yep, makes sense.